_includes/head.html

```diff
@@ -1,7 +1,7 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>{{ site.title }} - {% if page.title %}{{ page.title }}{% endif %}</title>
+  <title>{{ site.title }} {% if page.title %}{{ page.title }}{% endif %}</title>
```

Comment:
Thanks a lot for your contribution @ir4hul !

I'm afraid this will remove all hyphens and what we want instead, is to show it only when there is a `page.title` defined. Therefore, the hyphen should be moved within the `if` `endif` block, I believe.